jenkins/jobs/image-openwrt.yaml

```diff
@@ -11,6 +11,8 @@
         type: slave
         values:
         - amd64
+        - aarch64
+        - armv7l
```

Comment:
Those aren't valid architectures, we'd expect `arm64` and `armhf` but that will cause the image being built on the native architecture which from what you've mentioned over on the distrobuilder issue won't actually work.

Reply:
Should we then close this PR, and just have users build the images themselves with `distrobuilder`?

Reply:
We can merge the yaml change and just skip the jenkins change so they at least have a YAML they can use.

Reply:
Done.